helmfile.d/10-base.yaml

```diff
@@ -2,7 +2,9 @@ environments:
   default:
     values:
     - ../base.yaml
-    - ../production.yaml
+    - ../../etc/production.yaml
+    secrets:
+    - ../../secrets/production.yaml
```

Comment:
To streamline this, could we create a `environment.yaml` which contains references to these production / secrets files? This would not be committed to Git, although we can add a template. Or we need to add documentation to force all users to use these locations.

Reply:
Waiting to find a way to solve or workaround roboll/helmfile#1454

Reply:
Fixed in a3568f3